crates/test_utils/src/main.rs

```diff
@@ -0,0 +1,163 @@
+use std::{
+    env,
+    process::{exit, Command as cmd},
+};
+
+use clap::{arg, command, Parser};
+use router::types::ConnectorAuthType;
+use test_utils::connector_auth::ConnectorAuthenticationMap;
+
+// Just by the name of the connector, this function generates the name of the collection
+// Example: CONNECTOR_NAME="stripe" -> OUTPUT: postman/stripe.postman_collection.json
+fn path_generation(name: &String) -> String {
```

Comment:
```suggestion
fn path_generation(name: impl AsRef<str>) -> String {
```

This would allow you to accept any of `String`, `&String` and `&str`.

Also, consider annotating with `#[inline]`.